.github/workflows/multi-arch-build.yaml

```diff
@@ -87,25 +87,14 @@ jobs:
           echo "::set-output name=version::$VERSION"
 
       - name: Generate image FQIN(s) to push
-        id: gen_fqin
+        id: reponame_reg
         run: |
           if [[ "${{ matrix.source }}" == 'stable' ]]; then
             # The command version in image just built
             VERSION='v${{ steps.sniff_test.outputs.version }}'
             # workaround vim syntax-highlight bug: '
-            # Image tags previously pushed to quay
-            ALLTAGS=$(skopeo list-tags \
-                      docker://$REPONAME_QUAY_REGISTRY/stable | \
-                      jq -r '.Tags[]')
-
-            # New version? Push quay.io/$REPONAME/stable:vX.X.X and :latest
-            if ! fgrep -qx "$VERSION" <<<"$ALLTAGS"; then
-              # Assume version-tag is also the most up to date (i.e. "latest")
-              FQIN="$REPONAME_QUAY_REGISTRY/stable:$VERSION,$REPONAME_QUAY_REGISTRY/stable:latest"
-            else # Not a new version-tagged image
-              # Assume other contents changed, so this is the "new" latest.
-              FQIN="$REPONAME_QUAY_REGISTRY/stable:latest"
-            fi
+            # Push both new|updated version-tag and latest-tag FQINs
+            FQIN="$REPONAME_QUAY_REGISTRY/stable:$VERSION,$REPONAME_QUAY_REGISTRY/stable:latest"
```

Comment:
OK, long day of staring at a lot of zeroes and ones.  Should we be pushing both here?  I expected just `latest` unless the `$VERSION` variant wasn't present.

Reply:
AIUI the intention is to re-push all the time: even though our part isn't changing, underlying parts of the image itself may be updated upstream to (e.g.) address CVEs.

Reply:
That's correct.  In a future Cirrus-based re-implementation, I'd like to solve this in a slightly more elegant way.  For now, this is one of those "Best of the worst" alternatives :disappointed:

Reply:
Note: When we release a new version, the constant daily-updates of the (now) old version will stop.  So new CVEs will persist there for the joy and happiness of all :confused:

Reply:
Maybe we should chat this out at the water cooler tomorrow.  I only want to update versioned images when and if they contain a CVE fix.  If that means we need to do that by hand, I'm fine with that.

Reply:
I would rather automate it and not have humans involved.